de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/PlainInvoicingParams.java

```diff
@@ -52,6 +52,8 @@
 
 	private BigDecimal check_NetAmtToInvoice = null;
 
+	private boolean isUpdateLocationAndContactForInvoice = false;
```

Comment:
i think i'd prefer this to be renamed to `updateLocationAndContactForInvoice`..it better blends in with our general practice.
Also, then you can just annotate it with lombok's `@Getter` if you want and save the 4 lines  further down.

Reply:
done